frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/SelectEmbedTypePane.unit.spec.tsx

```diff
@@ -81,7 +84,10 @@ describe("SelectEmbedTypePane", () => {
 
     describe("when the resource is not published", () => {
       it("should call `goToNextStep` with `application` when the static embedding option is clicked", async () => {
-        const { goToNextStep } = setup({ isResourcePublished: true });
+        const { goToNextStep } = setup({
+          isResourcePublished: false,
```

Comment:
I believe this was a mistake since the described name stated that the resource shouldn't be published, otherwise, we just basically ran the same test 2 times. This change make it so that it test different scenario from the test above.